torch/testing/_internal/common_methods_invocations.py

```diff
@@ -39,6 +39,10 @@
 import torch._refs.special
 import torch._refs.linalg
 
+# Make sure that decompositions used for test_forward_mode_AD and
+# test_fn_fwgrad_bwgrad are registered to the jit
+import torch._decomp.decompositions_for_jvp
+
```

Comment:
Hold on: when do these decompositions get registered outside of testing? It's a bit weird to require users to call `import torch._decomp.decompositions_for_jvp` before they can use the decompositions

Reply:
Good catch, it needs to be imported somewhere outside of testing. I keep forgetting to add this.